arcgis-ios-sdk-samples/Layers/Blend renderer/BlendRenderer.storyboard

```diff
@@ -74,117 +74,167 @@
         <scene sceneID="zuR-Aq-sif">
```

Comment:
This scene is reporting the following errors:

![screen shot 2018-10-19 at 3 53 23 pm](https://user-images.githubusercontent.com/2257493/47247212-2818f400-d3b7-11e8-83f8-0b58747a7af3.png)

The problem is that the provided constraints specify a *minimum* height for the cell, not an *exact* height. The top and bottom constraints should be equal to (not greater than or equal to) and the centerY constraint should be removed.

Reply:
Thanks for catching this! Not sure how I missed the error.